src/main/java/io/anserini/rerank/ScoredDocuments.java

```diff
@@ -126,4 +131,36 @@ public static ScoredDocuments fromESDocs(SearchHits rs) {
 
     return scoredDocs;
   }
+
+  public static ScoredDocuments fromRelDocs(Map<String, Integer> queryRelDocs, IndexReader reader) throws IOException {
+    ScoredDocuments scoredDocs = new ScoredDocuments();
```

Comment:
How about `fromQrels`? and `queryRelDocs` -> `qrels`?

Reply:
I have `Rel` here and other places to indicate that during the process there are only relevant documents.  I think it would be better to keep that in place?

Reply:
Don't you need non-relevant documents for BM25?

Reply:
Yes, but RF only affects the feedback methods such as rm3, and we only need relevant documents for that?

Reply:
So shouldn't we load all the qrels, and each technique can ignore the parts they don't need? E.g., RM3 ignores all the rel grade = 0.

In terms of future proofing, we probably want to add an option of what's rel in the future?
E.g., in multi-grade, just 2 is rel, or both 1+2 are rel?

So loading all qrels seems more general?

Reply:
Sure, sg.